src/quo2/components/avatars/icon_avatar.cljs

```diff
@@ -26,5 +26,7 @@
               :justify-content  :center
               :align-items      :center}}
      [icons/icon icon
-      {:size  icon-size
-       :color icon-color}]]))
+      {:size            icon-size
+       :container-style {:width  20
```

Comment:
could you elaborate on why :container-style is needed here?

Reply:
icon has a size prop, width shouldn't be used

Reply:
What if we wanted to configure that icon's container's dimensions?

Reply:
margins ?

Reply:
Margins control spacing I meant the container's sizing.

Reply:
i'm not sure i understand what is container's sizing, why?

Reply:
I mean the sizing of the container that's around the icon, It's parent container.

Reply:
but why margin or padding can't be used ?

Reply:
```
[rn/image
       {:style
        (merge {:width  size
                :height size}

               (when (not no-color)
                 {:tint-color (if (and (string? color) (not (string/blank? color)))
                                color
                                (colors/theme-colors colors/neutral-100 colors/white theme))})

               container-style)
```
so width and height managed by size prop, overriding them by ` container-style` is wrong, different size should be used instead

Reply:
do we have a component? if so there should be an icon size prop, if sometimes designers want a bigger icon, i believe it's only for some specific cases, so currently default size works fine for most cases

Reply:
Will post side by side comparison between using what's already there and what's proposed.

Reply:
Fixed, I had to adjust icon-avatar, Thanks for your comments!

Reply:
Kindyl take another look